src/core/sys/freebsd/sys/event.d

```diff
@@ -43,15 +44,33 @@ extern(D) void EV_SET(kevent_t* kevp, typeof(kevent_t.tupleof) args)
     *kevp = kevent_t(args);
 }
 
-struct kevent_t
+static if (__FreeBSD_version >= 1200000 && __FreeBSD_version < 1300000)
```

Comment:
I'd be more optimistic and use `__FreeBSD_version >= 1200000` alone. If FreeBSD 13 wouldn't change any relevant struct, it'd just work without having to touch druntime. FreeBSD changes would have to be tracked anyway, as it could just as easily change a struct which hasn't been versioned already.

Reply:
Done in the third commit.